bokun.yaml

```diff
@@ -5688,56 +5692,73 @@ definitions:
       rateId:
         format: int64
         type: integer
+        description: 'Specifies the rate ID.'
       title:
         type: string
+        description: 'Specifies the title of the rate.'
       price:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the rate price if it is set per booking. Field omitted if the price is set per person.'
       passengers:
         items:
           $ref: '#/definitions/CategoryPriceDto'
         type: array
+        description: 'The list of pricing categories and related prices per person. This field is omitted if the price is set per booking.'
       pickupPrice:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the price for the pick up service on this rate.'
       dropoffPrice:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the price for the drop-off service on this rate.'
       extras:
         items:
           $ref: '#/definitions/ExtraPriceDto'
         type: array
+        description: 'The list of prices for the extras available for this rate.'
   ActivityPriceListDateRange:
     type: object
     properties:
       from:
         type: string
+        description: 'Start date for the validity of this pricelist.'
```

Comment:
Please include if this date is inclusive or exclusive as it matters a lot when speaking of ranges. Also worth mentioning whether this is optional or mandatory field.